### ISSUE TITLE
Compilation error: Failure to find com.compomics:utilities:jar:4.12.0-SNAPSHOT in http://genesis.UGent.be/maven2 was cached in the local repository

### ISSUE THREAD

**Issue by lcscs12345** [compomics/searchgui#153]
Hi,

Thank you for creating such a useful proteomics tool. I would like to run Crux Tide in CLI without having to reindex the fasta for every single run. Because I couldn't find a way to avoid reindexing, I tried to change the --overwrite option and compile searchgui by
`mvn -e -X compile`

However, the compilation failed and here is the log file. Any suggestions? Thank you!
[compile.txt.zip](https://github.com/compomics/searchgui/files/1220163/compile.txt.zip)

ps - to avoid reindexing, found a workaround by breaking crux tide-index using an invalid indexing parameter.

**Comment by mvaudel** [comment]
Hi,

Apologies for this, we took advantage that there is less usage in July-August to start some major refactoring of our code. We are currently having some unresolved issues after merging branches, I will come back to you as soon as things are working again.

Apologies for the inconvenience,

Marc

**Comment by hbarsnes** [comment]
Hi,

utilities 4.12.0 has now finally been deployed. Meaning that the master branch of SearchGUI should compile again without requiring the unavailable snapshot. However, we strongly recommend sticking with the released versions until we've completed the major refactoring mentioned above.

But if you do play with the SearchGUI code we recommend starting from this branch: https://github.com/compomics/searchgui/tree/new_backend.

Best regards,
Harald

PS: I will now close this issue, as the problem stated in the issue title has been resolved, but don't hesitate to open a new one if you have other questions.
